plugwise/helper.py

```diff
@@ -276,109 +341,56 @@ def all_appliances(self):
             self.active_device_present = True
 
         for appliance in self._appliances:
-            appliance_location = None
-            appliance_types = set()
-
-            appliance_id = appliance.attrib["id"]
-            appliance_class = appliance.find("type").text
-            appliance_name = appliance.find("name").text
-            appliance_model = appliance_class.replace("_", " ").title()
-            appliance_fw = None
-            appliance_v_name = None
-
+            appl = Munch()
```

Comment:
Nice! First time I hear about Munch() :)

Reply:
Me too :) But it makes for much more object-oriented approach and less fugly dict-approach (and I really like it so far)

Reply:
:+1: